run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -eu
+
+INITI_DATA= [ ! -f db.sqlite3 ]
+
+./manage.py makemigrations
+./manage.py migrate
+
+if $INIT_DATA ; then
```

Comment:
there's a typo with `INITI_DATA`, but we can just write this as
```bash
if [ ! -f db.sqlite3 ]; then
```

Reply:
Thanks for the catch.

We actually can't since the file will be created by the migration command, it has to be detected beforehand